Live/LiveEventWithDVR/Program.cs

```diff
@@ -1,663 +1,562 @@
-//  8) Get the preview endpoint to monitor in a player for DASH or HLS.
-//  9) Get the ingest RTMP endpoint URL for use in OBS Studio.
-//     Set up OBS studio and start the broadcast.  Monitor the stream in 
-//     your DASH or HLS player of choice. 
-// 10) Create a new Streaming Locator on the recording Asset object from step 5.
-// 11) Get the URLs for the HLS and DASH manifest to share with your audience
-//     or CMS system. This can also be created earlier after step 5 if desired.
-////////////////////////////////////////////////////////////////////////////////////
-
-namespace LiveEventWithDVR
+//////////////////////////////////////////////////////////////////////////////////////
+////  Azure Media Services Live streaming sample 
+
+////  This sample assumes that you will use OBS Studio to broadcast RTMP
+////  to the ingest endpoint. Please install OBS Studio first. 
+////  Use the following settings in OBS:
+////      Encoder: NVIDIA NVENC (if avail) or x264
+////      Rate Control: CBR
+////      Bitrate: 2500 Kbps (or something reasonable for your laptop)
+////      Keyframe Interval : 2s, or 1s for low latency  
+////      Preset : Low-latency Quality or Performance (NVENC) or "veryfast" using x264
+////      Profile: high
+////      GPU: 0 (Auto)
+////      Max B-frames: 2
+////      
+////  The workflow for the sample and for the recommended use of the Live API:
+////  1) Create the client for AMS using AAD service principal or managed ID
+////  2) Set up your IP restriction allow objects for ingest and preview
+////  3) Configure the Live Event object with your settings. Choose pass-through
+////     or encoding channel type and size (720p or 1080p)
+////  4) Create the Live Event without starting it
+////  5) Create an Asset to be used for recording the live stream into
+////  6) Create a Live Output, which acts as the "recorder" to record into the
+////     Asset (which is like the tape in the recorder).
+////  7) Start the Live Event - this can take a little bit.
+////  8) Get the preview endpoint to monitor in a player for DASH or HLS.
+////  9) Get the ingest RTMP endpoint URL for use in OBS Studio.
+////     Set up OBS studio and start the broadcast.  Monitor the stream in 
+////     your DASH or HLS player of choice. 
+//// 10) Create a new Streaming Locator on the recording Asset object from step 5.
+//// 11) Get the URLs for the HLS and DASH manifest to share with your audience
+////     or CMS system. This can also be created earlier after step 5 if desired.
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+// Event Hub config
+string StorageAccountName = "---set-your-storage-account-name-here---";
+string StorageAccountKey = "---set-your-storage-account-key-here---";
+string blobContainerName = "---set-your-blob-container-name-here---";
+string eventHubsConnectionString = "---set-your-event-hubs-connection-string-here---";
```

Comment:
@jonpayne  do you want to also remove Event Hubs from our live samples? I think it is a little trickier to remove for some of the events like Encoder connect and disconnect here.

Reply:
We probably need it for now for the events you mentioned. It might be good to replace it with a Service Bus queue eventually (that should require less setup).